.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 dist
-example/package-lock.json
\ No newline at end of file
+example/package-lock.json
+package-lock.json
```

Comment:
I'm confused, don't we want to check in the package-lock.json?

Reply:
No until we start using it, package-lock.json is only used when you run "npm ci", which we don't do. Maybe if we start using TravisCI we can put it back.